docs/src/main/asciidoc/security-iap.adoc

```diff
@@ -13,8 +13,12 @@ The following claims are validated automatically:
  * Expiration time
  * Issuer
 
-Additionally, Audience (`"aud"`) validation is enabled when the application is running on AppEngine/ComputeEngine/Kubernetes, or when a custom audience is provided through `spring.cloud.gcp.security.iap.audience` property.
-The custom property overrides automatic environment-based checks.
+Additionally, Audience (`"aud"`) validation is enabled when the application is running on AppEngine, or when a custom audience is provided through `spring.cloud.gcp.security.iap.audience` property.
```

Comment:
Isn't audience validation always enabled? It's just that the audience is automatically detected on App Engine, but not on GCE/GKE.